tests/Form/FormMapperTest.php

```diff
@@ -494,6 +494,16 @@ public function testGroupRemovingWithTabAndWithTabRemoving(): void
         $this->assertSame([], $this->admin->getFormTabs());
     }
 
+    public function testTabRemoving(): void
+    {
+        $this->formMapper->tab('mytab')->with('foobar');
```

Comment:
Shall we assert first that it is set?

Reply:
tab/with methods are already tested in another test